src/gameplay/types.ts

```diff
@@ -43,7 +43,7 @@ export interface IUser {
   emailTokenExpiry?: Date;
```

Comment:
Let's add that automated check to limit number of avatar requests if they've hit their max limit of library stuff.